lib/api/attributes.js

```diff
@@ -73,6 +73,71 @@ exports.attr = function(name, value) {
   return getAttr(this[0], name);
 };
 
+var getProp = function (el, name) {
+  return el.hasOwnProperty(name)
+      ? el[name]
+      : rboolean.test(name)
+          ? getAttr(el, name) !== undefined
+          : getAttr(el, name);
+};
+
+var setProp = function (el, name, value) {
+  el[name] = rboolean.test(name) ? !!value : value;
+};
+
+exports.prop = function (name, value) {
+  var i = 0,
+      property;
+
+  if (typeof name === 'string' && value === undefined) {
+
+    switch (name) {
+      case 'style':
+        property = this.css();
+
+        _.each(property, function (v, p) {
```

Comment:
If you prefer, I can remove the setting of property names and length on the property object...  Or to go even further, remove the special handling of getting 'style' and 'tagName|nodeName' altogether.  Any of those options would be reasonable in order to get a prop() method.